plot/analyze-specs.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys, os, shutil, re, argparse, json, glob
+from codecs import open
+from itertools import izip
+from collections import defaultdict, Counter, OrderedDict
+
```

Comment:
documentation saying that this reads spec, generates initial-templates, which gets consumed by VegaRandomizer is helpful.

Reply:
Done